lib/Basic/Subprocess.cpp

```diff
@@ -207,18 +219,32 @@ class ControlProtocolState {
 // Helper function to collect subprocess output
 static void captureExecutedProcessOutput(
     ProcessDelegate& delegate,
-    int outputPipe,
+    FD outputPipe,
     ProcessHandle handle,
     ProcessContext* ctx
 ) {
-  // Read all the data from the output pipe.
   while (true) {
     char buf[4096];
+#if defined(_WIN32)
+    DWORD numBytes;
+    if (!ReadFile(outputPipe, buf, sizeof(buf), &numBytes, nullptr)){
+      int err = GetLastError();
+      FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM,
```

Comment:
Please use FormatMessageW in order to support Unicode.

Also, please define `UNICODE` and `_UNICODE` somewhere centrally, to avoid any risk of accidentally using the ASCII functions.